data/tools.yaml

```diff
@@ -278,3 +278,9 @@
   tags:
     - parsers
     - renderers
+- name: docprint
+  summary: Generate Clean Static HTML documentation from API Blueprint file. Provides waypoints, attributes and multiple language code snippets.
+  url: https://github.com/swathysubhash/docprint
+  tags:
+    - parsers
```

Comment:
Hi @swathysubhash.

Thanks for contributing a new tool. Would you be able to remove the `parsers` tag as this tool isn't a parser and it uses `drafter` to parse API Blueprints.

Once that is changed this is good to go.

Thanks